app/lib/submission_builder/ty2024/states/md/documents/md502.rb

```diff
@@ -178,10 +179,6 @@ def has_exemptions?
     has_dependent_exemption? || has_line_a_or_b_exemptions
   end
 
-  def has_state_tax_computation?
-    @deduction_method_is_standard || calculated_fields.fetch(:MD502_LINE_22)&.positive?
-  end
-
```

Comment:
paired with Em on taking this logic out now that we have more fields in the tax computation section that will need to be visible at all times